src/Drawer/Drawer.js

```diff
@@ -164,10 +383,13 @@ class Drawer extends React.Component {
     return (
       <Modal
         BackdropProps={{
+          ref: ref => {
+            this.backdrop = ref != null ? ReactDOM.findDOMNode(ref) : null;
+          },
           transitionDuration,
         }}
         className={classNames(classes.modal, className)}
-        open={open}
+        open={open || (type === 'temporary' && !!this.maybeSwiping)}
```

Comment:
Why use !! notation when maybeSwiping is already a boolean value

Reply:
`this.maybeSwiping` may be undefined before swiping the first time